app/code/Bulbulatory/Recommendations/Model/RecommendationRepository.php

```diff
@@ -36,6 +50,65 @@ public function __construct(ResourceConnection $resourceConnection)
      */
     public function deleteByIds(array $ids)
     {
-        return $this->connection->delete(Recommendation::MAIN_TABLE, 'id IN (' . implode(',', $ids) . ')');
+        return $this->connection->delete(RecommendationResource::MAIN_TABLE, 'id IN (' . implode(',', $ids) . ')');
+    }
+
+    /**
+     * Checking if recommendation for given customer and recommended email already exist in database
+     * @param Customer $customer
+     * @param string $recommendedEmail
+     * @return bool
+     */
+    public function isRecommendationExist(Customer $customer, string $recommendedEmail)
+    {
+
+        $select = $this->connection->select()
+            ->from(RecommendationResource::MAIN_TABLE)
+            ->columns('COUNT(*) as count')
+            ->where('customer_id = ' . $customer->getId() . ' AND email = "' . $recommendedEmail . '"');
```

Comment:
Magento pozwala używać placeholderów w takich zapytaniach:
```suggestion
            ->where('customer_id = ?', $customer->getId())
            ->where('email = ?', $recommendedEmail');
```
Wtedy też automatycznie te wartości są escape'owane.